File: workspace_presentacion/Flotix2021/Flotix2021/Model/Vehiculo.cs

namespace Flotix2021.Model
{
    /// <summary>
    /// Objeto Vehiculo para mandar al servidor
    /// </summary>
    class Vehiculo
    {
        public string matricula { get; set; }
        public string fechaMatriculacion { get; set; }
        public string modelo { get; set; }
        public int plazas { get; set; }
        public int capacidad { get; set; }
        public int km { get; set; }
        public bool disponibilidad { get; set; }
        public bool baja { get; set; }
        public string nombreImagen { get; set; }
        public string nombreImagenPermiso { get; set; }
    }
}

File: workspace_movil/app/src/main/java/com/flotix/model/Vehiculo.kt
package com.flotix.model

import java.io.Serializable

class Vehiculo(
    var matricula: String = "",
    var fechaMatriculacion: String = "",
    var modelo: String = "",
    var plazas: Int = 0,
    var capacidad: Int = 0,
    // TODO permiso de circulacion
    var km: Int = 0,
    var disponibilidad: Boolean = false,
    var baja: Boolean = false
) : Serializable {
}